plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieVersionTransformer.java

```diff
@@ -381,6 +384,26 @@ private static KieProcessFormField buildField(  String id, String modelName,
 
         }
         
+        if (field.getType().equals("RadioGroup")) {
```

Comment:
Not specific to this PR but we need to add logging for the case where we encounter a fieldType we don't recognize. Should be a WARN